docs/pages/kubernetes-access/getting-started/cluster.mdx

```diff
@@ -131,8 +133,7 @@ for web apps using [application access](../../application-access/introduction.md
 
     # Create a JSON file changeset for AWS.
     $ jq -n --arg ip ${MYIP?} --arg dns ${MYDNS?} '{"Comment": "Create records", "Changes": [
-      {"Action": "CREATE", "ResourceRecordSet": {"Name": $dns, "Type": "A", "TTL": 300, "ResourceRecords": [{ "Value": $ip}]}},
-      {"Action": "CREATE", "ResourceRecordSet": {"Name": ("*." + $dns), "Type": "A", "TTL": 300, "ResourceRecords": [{ "Value": $ip}]}}
```

Comment:
I'm not sure why are we removing the commands for creating wildcard records? If we do that, it is no longer a copy-pasteable example :) Same below.

Reply:
Good point—I've restored the original commands while adding a reminder to adjust them for your environment in case, say, a user doesn't want to create a wildcard record.